### PR TITLE
Add terms and conditions page + notice to the signup page

### PR DESCRIPTION
**Pull request by pcraig3**
This PR

- Adds a "Terms and conditions" page to the app
- Puts a "Terms" link in the footer
- Adds a notice to the bottom of the signup form
- Translates the remaining un-translated field labels on the signup form
- Adds french translations for them all

[Trello card for this story](https://trello.com/c/IjBHKVid/242-people-agree-to-terms-of-use-by-creating-an-account).

## Screenshots

| Terms page | Terms notice (on signup form) |
|--------|-------|
|  Screen Shot 2020-08-10 at 9 12 07 AM  |   image   |

### CODE REVIEW

File: locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1005,25 +1018,85 @@ msgstr ""
...
+"modalités d’utilisation, ou pour signaler un problème technique, écrivez à "
+"<a href=\"mailto:assistance@cds-snc.ca\">assistance@cds-snc.ca</a>.</p>"
+
+#: profiles/validators.py:10 profiles/validators.py:22
+msgid "Your password can’t be a commonly-used sequence."
+msgstr "Votre mot de passe ne peut pas être utilisé couramment."
 
 #: profiles/validators.py:17
 msgid "Your password can’t contain terms related to COVID-19"
 msgstr ""
```
Comment (PierrePaul) [inline locale/fr/LC_MESSAGES/django.po:1098]: Missing translation here

Comment (pcraig3) [reply locale/fr/LC_MESSAGES/django.po:1098]: Is this a good translation for this?

> Votre mot de passe ne peut pas contenir de termes liés à COVID-19

Comment (PierrePaul) [reply locale/fr/LC_MESSAGES/django.po:1098]: sounds good!

File: locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1005,25 +1018,85 @@ msgstr ""
...
+"ul><h2>Utilisation du portail</h2><p>Vous devez:</p><ul><li>Suivre toutes "
+"les procédures relatives au portail établies par votre autorité sanitaire.</"
+"li><li>Fournir des clés à usage unique seulement aux patients ayant un "
+"diagnostic de COVID-19 établi par votre autorité sanitaire.</li><li>Prendre "
+"les mesures nécessaires pour empêcher toute personne, y compris celles "
+"travaillant pour une autorité sanitaire, de découvrir votre nom "
+"d’utilisateur et votre mot de passe.</li></ul><h2>Signalement des activités "
+"non autorisées</h2><p>Vous devez :</p><ul><li>Signaler les activités non "
+"autorisées, la perte d’un nom d’utilisateur ou d’un mot de passe, ou tout "
+"autre problème de sécurité à <ahref=\"mailto:security@cds-snc.ca"
```
Comment (PierrePaul) [inline locale/fr/LC_MESSAGES/django.po:1085]: Missing space in `<ahref`

**Pull request merged**